src/platform/forms/save-in-progress/RoutedSavableApp.jsx

```diff
@@ -51,15 +51,21 @@ class RoutedSavableApp extends React.Component {
     //  saved form / prefill
     // If we're in production, we'll redirect if we start in the middle of a form
     // In development, we won't redirect unless we append the URL with `?redirect`
-    const { currentLocation } = this.props;
+    const { currentLocation, formConfig } = this.props;
+    const { additionalRoutes = [] } = formConfig;
+    const additionalSafePaths =
+      additionalRoutes && additionalRoutes.map(route => route.path);
     const trimmedPathname = currentLocation.pathname.replace(/\/$/, '');
     const resumeForm = trimmedPathname.endsWith('resume');
     const devRedirect =
       (!environment.isLocalhost() &&
         !currentLocation.search.includes('skip')) ||
       currentLocation.search.includes('redirect');
     const goToStartPage = resumeForm || devRedirect;
-    if (isInProgressPath(currentLocation.pathname) && goToStartPage) {
+    if (
+      isInProgressPath(currentLocation.pathname, additionalSafePaths) &&
```

Comment:
Mirroring the same logic we used [here](https://github.com/department-of-veterans-affairs/vets-website/pull/10010) to determine if the current path is actually a page _within_ the form. With this change, the `id-form` path will be considered a non-form route so a logged-in user with an in-progress HCA will not be redirected to where the left off in the form.